ui/components/__tests__/DataTable.test.tsx

```diff
@@ -62,7 +62,7 @@ describe("DataTable", () => {
   describe("sorting", () => {
     describe("sortWithType", () => {
       it("should handle sorting with case SortType.string", () => {
-        const nameSort = sortWithType(rows, {
+        const nameSort = sortWithType(rows, false, {
```

Comment:
We can probably remove these tests? They are just testing lodash functionality now, or? wdyt?

Reply:
yes we can do that they are testing lodash :D